src/main/java/org/alliancegenome/curation_api/services/GeneExpressionAnnotationService.java

```diff
@@ -18,17 +20,22 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
+@JBossLog
```

Comment:
Remove unused JBossLog annotation and import

Reply:
Removed